python-package/securexgboost/rabit.py

```diff
@@ -22,16 +26,35 @@ def _init_rabit():
 
 def init(args=None):
     """Initialize the rabit library with arguments"""
-    if args is None:
-        args = []
-    arr = (ctypes.c_char_p * len(args))()
-    arr[:] = args
-    _LIB.RabitInit(len(arr), arr)
+
+    core_globals = pass_globals()
```

Comment:
Are globals not available across modules?

Reply:
I tried, but I couldn't get it to work without doing something like this 😞